src/main/java/com/example/petbuddybackend/service/offer/OfferService.java

```diff
@@ -150,10 +176,13 @@ private List<OfferConfiguration> createAdditionalConfigurationsForOffer(List<Mod
         List<OfferConfiguration> newOfferConfigurations = new ArrayList<>();
         for(ModifyConfigurationDTO offerConfiguration : offerConfigurations) {
             OfferConfiguration configuration = createConfiguration(offerConfiguration, offer);
-            checkForDuplicateConfiguration(Stream.concat(
-                    Optional.ofNullable(offer.getOfferConfigurations()).orElse(Collections.emptyList()).stream(),
-                    newOfferConfigurations.stream()
-            ).toList(), configuration);
+            checkForDuplicateConfiguration(
+                    Stream.concat(
+                        Optional.ofNullable(offer.getOfferConfigurations()).orElse(Collections.emptyList()).stream(),
```

Comment:
W sumie offer to jest entity, więc zamiast Optional.ofNullable to zsetować pustą listę w entity?

Reply:
Z resztą widzę że już jest zsetowane

Reply:
Tak, między pierwszym pisaniem tego kodu były zmiany że kolekcje są setowane od razu więc odświeże to potem troche